configurator/src/MasterConfigurationPage.ui

```diff
@@ -185,6 +185,11 @@
               <string>Only user name</string>
              </property>
             </item>
+            <item>
+             <property name="text">
+              <string>Only last part of user name (hopefully surname)</string>
```

Comment:
Please change to "Last part of user name only" and remove the "(hopefully surname)".